state/allwatcher_internal_test.go

```diff
@@ -4241,7 +4261,7 @@ func (tw *testWatcher) AssertNoChange(c *gc.C) {
 func (tw *testWatcher) AssertChanges(c *gc.C, expected int) {
 	var count int
 	tw.st.StartSync()
-	maxWait := tw.st.clock().After(testing.LongWait)
+	maxWait := time.After(testing.LongWait)
```

Comment:
good catch. (since the point is to avoid having the test wait indefinitely, using a clock doesn't get us that.)